docs/dapps/getting-started.md

```diff
@@ -8,91 +8,531 @@ import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
+}
+```
+
+You can now run the test:
+
+```sh
 npx @alephium/cli@latest test
 ```
 
+Congratulations, you should see what result would produce a withdraw.
+
 ## Deploy your contract
 
-Next, to deploy the contract we will use Alephium CLI and a deployment script `scripts/0_deploy_faucet.ts`. You can run it using:
+Now things are getting serious, we will deploy our contract on our `devnet` :rocket:
+
+The `deploy` command will deploy everything which is inside the `scripts` folder.
 
+```sh
+mkdir scripts
 ```
+
+Let's create this file into this folder: `scripts/0_deploy_faucet.ts`
+Note that deployment scripts should prefixed with numbers (starting from 0)
+
+```typescript
+import { Deployer, DeployFunction, Network } from '@alephium/cli'
+import { Settings } from '../alephium.config'
+import { TokenFaucet } from '../artifacts/ts'
+
+// This deploy function will be called by cli deployment tool automatically
+// Note that deployment scripts should prefixed with numbers (starting from 0)
+const deployFaucet: DeployFunction<Settings> = async (
+  deployer: Deployer
+): Promise<void> => {
+  const issueTokenAmount = 100n
+  const result = await deployer.deployContract(TokenFaucet, {
+    // The amount of token to be issued
+    issueTokenAmount: issueTokenAmount,
+    // The initial states of the faucet contract
+    initialFields: {
+      symbol: Buffer.from('TF', 'utf8').toString('hex'),
+      name: Buffer.from('TokenFaucet', 'utf8').toString('hex'),
+      decimals: 18n,
+      supply: issueTokenAmount,
+      balance: issueTokenAmount
+    }
+  })
+  console.log('Token faucet contract id: ' + result.contractId)
+  console.log('Token faucet contract address: ' + result.contractAddress)
+}
+
+export default deployFaucet
+```
+
+[deployContract](https://github.com/alephium/alephium-web3/blob/d2b5b63cae015e843aa77b4cf484bc62a070f1d5/packages/cli/src/types.ts#L133-L137) takes our contract and deploy it with the correct arguments, you can also add a `taskTag` arguments, to tag your deployment with a specific name. By default it will use the contract name, but if you deploy multiple time the same contract with different initial fields, your `.deployement` file will get override, using a specific `taskTag` solve this issue.
+
+From the [DeployContractParams](https://github.com/alephium/alephium-web3/blob/d2b5b63cae015e843aa77b4cf484bc62a070f1d5/packages/web3/src/contract/contract.ts#L1286-L1293) interface, we can see that `initialFields` is mandatory as it contains the arguments for our `TokenFaucet` contract.
+
+With `issueTokenAmount` you can define how many tokens you want to issue, this is required if you want to create a token, otherwise no token-id will be created.
+
+Now let's deploy!!!
+
+```sh
 npx @alephium/cli@latest deploy
 ```
 
-This will deploy the token faucet to all of the 4 groups of Devnet. You could configure `alephium.config.ts` to deploy the contract to different networks.
+...OOPS... It doesn't work???
+
+`NO UTXO found` ???
```

Comment:
I got a different error...

```
Loading alephium config file: /Users/ilias/dev/alephium/_test-projects/alephium-faucet-tuto/alephium.config.ts
Failed to deploy the project
Failed to deploy contracts, error: Error: The node chain id 4 is different from configured chain id 2
    at validateChainParams (/Users/ilias/.npm/_npx/f1b120e1b513c04c/node_modules/@alephium/cli/src/deployment.ts:408:11)
```

I guess this is because I am using the `alephium-stack` project to run the devnet with Docker, which sets the network id to be `4`: https://github.com/alephium/alephium-stack/blob/master/devnet/docker-compose.yml#L100

Reply:
HAHA, after I changed the network id in my `alephium.config.ts` file to `4`, I didn't get this UTXO error that you mentioned!

```console
➜  alephium-faucet-tuto npx @alephium/cli@latest deploy
Loading alephium config file: /Users/ilias/dev/alephium/_test-projects/alephium-faucet-tuto/alephium.config.ts
Contracts are compiled already. Loading them from folder "artifacts"
Deploying contract TokenFaucet
Deployer - group 0 - 1DrDyTr9RpRsQnDnXo2YRiPzPW4ooHX5LLoqXrqfMrpQH
Token faucet contract id: 7fd8acfda8f2769f52eb7536443520ae1dcaec80d480d7cb25337db36c007a00
Token faucet contract address: 23J1X5enQyiJ2swkzXCSKqhkp6dBCV7kgEYiuPTLJnXBD
✅ Deployment scripts executed!
```